cmd/dra-example-kubeletplugin/cdi.go

```diff
@@ -89,18 +89,39 @@ func (cdi *CDIHandler) CreateClaimSpecFile(claimUID string, devices PreparedDevi
 		Devices: []cdispec.Device{},
 	}
 
-	gpuIdx := 0
-	for _, device := range devices {
+	for i, device := range devices {
+		envs := []string{
+			fmt.Sprintf("GPU_DEVICE_%d=%s", i, device.DeviceName),
+		}
+
+		if device.Config.Sharing != nil {
+			envs = append(envs, fmt.Sprintf("GPU_SHARING_STRATEGY=%s", device.Config.Sharing.Strategy))
+		}
```

Comment:
Does it make sense when different devices configure different strategies?

The variable can only be set to one value, so perhaps not?

This is of course just a hypothetical example, but it should still handle corner cases in a sane way.

Reply:
Same below.

Reply:
Are you suggesting to have one variable per device since different devices might have different sharing strategiesß Yes, that makes sense. I will make the change.

Reply:
Updated.